includes/templates/template-single.php

```diff
@@ -73,7 +71,7 @@
                         </article>
 
                     <?php endwhile; ?>
-
```

Comment:
getting ready to add the per page feedback widget, I would put the custom element in here right?

Reply:
nope - do a search for PER PAGE FEEDBACK & you'll find where it goes